ci/create-new-project

```diff
@@ -1,14 +1,10 @@
 #!/bin/bash -ex
 
 TAG=$(ci/release-tag)
-docker build --target=builder --tag=camptocamp/geomapfish-build:${TAG} \
-    --build-arg=MAJOR_VERSION=${MAJOR_VERSION} --build-arg=VERSION=${TAG} .
-docker build --target=runner --tag=camptocamp/geomapfish:${TAG} \
-    --build-arg=MAJOR_VERSION=${MAJOR_VERSION} --build-arg=VERSION=${TAG} .
-docker build --target=upgrader --tag=camptocamp/geomapfish-scaffolds:${TAG} \
-    --build-arg=MAJOR_VERSION=${MAJOR_VERSION} --build-arg=VERSION=${TAG} .
-docker build --tag=camptocamp/geomapfish-config-build:${TAG} \
-    --build-arg=VERSION=${TAG} docker/config
+VERSION=${TAG} DOCKER_TAG=${TAG} make build-tools
+VERSION=${TAG} DOCKER_TAG=${TAG} make build-runner
+VERSION=${TAG} DOCKER_TAG=${TAG} make build-config-build
```

Comment:
Not already done in ci ?

Reply:
No because TAG is a calculated value (from MAJOR_VERSION, branch name and git tag)

Reply:
No, because it's dynamically generated ...